### DISCUSSION TITLE
Unable to find emitter for node type: XamlX.Ast.XamlAstXmlDirective

### DISCUSSION THREAD

**Discussion by BHaesen** [AvaloniaUI/Avalonia#12723] [Q&A]
I created an Avalonia project with version 0.10 and upgraded to version 11.
In 0.10 the preview of an axaml file was working fine.
In 11 I get the above message when opening an axaml file in the Error List and the preview is not shown.

I'm running VS 2022 17.7.3.
Just downloaded the Avalonia Extensions and restarted VS, to be sure.

**Comment by timunie** [comment]
@BHaesen if opening issues, fill in the templates provided. 

Also make sure to read migration guide. There are many breaking changes, just updating the nuget is not enough.

**Reply by timunie** [reply]
converted to a Q&A for now as I think this is not an actual bug but a question about upgrading.

**Comment by BHaesen** [comment]
I did follow the upgrade comments in https://docs.avaloniaui.net/docs/next/stay-up-to-date/upgrade-from-0.10 but this did not work.
I ended up re-creating the project in 11.0 and that worked. Fortunately it was a relative small project.

**Reply by timunie** [reply]
sorry to hear that. I know updating to 11.0 can be kind of heavy lifting, but it is worth it. So many improvements on board ^^
